packages/react/src/components/FileUploader/FileUploaderButton.js

```diff
@@ -78,17 +77,20 @@ function FileUploaderButton({
 
   return (
     <>
-      {/* eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions */}
-      <label
-        ref={innerRef}
-        tabIndex={disabled ? -1 : tabIndex || 0}
```

Comment:
Should we deprecate `tabIndex` since it's no longer explicitly used like this? With the new setup `tabIndex` will be passed through `...other` onto `<button>`, right?

If we deprecate, what would the message be? They technically can still pass a `tabIndex`, it's just won't be included in the prop types anymore.